kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -6,13 +6,35 @@ class ErrorReport {
   getErrorReport() {
     throw new Error('getErrorReport() method must be implemented.');
   }
+  getDeviceInfo() {
+    return {
+      type: /Mobi|Android/i.test(navigator.userAgent) ? 'Mobile' : 'Desktop',
```

Comment:
This will return the wrong `type` if kolibri is used on a tablet for example.

Reply:
(Just noting that this is quite an unreliable way to check for device type)

Reply:
Yes, definitely just relying on the values already defined in browserInfo.js seems like the right way forward here, no need to write new code when we already have robust parsing of the userAgent.